rest-api/src/main/resources/definitions/request_params.yml

```diff
@@ -114,6 +117,12 @@ properties:
     predicate:
         $ref: ./enums/search_term_predicate.yml
         readOnly: true
+    progressionFilters:
+        type: array
+        items: 
+            $ref: ./enums/participant_study_progress.yml
+            type: string
+        readOnly: true
```

Comment:
Same as previous comment, just noting the other location of this filter.

Reply:
Changed on server and now matches.